Makefile

```diff
@@ -15,7 +15,12 @@ LINT_TARGETS=\
 	pcari/apps.py\
 	pcari/signals.py\
 	pcari/urls.py\
-	pcari/views.py
+	pcari/views.py\
```

Comment:
BTW are we able to add any other files to the linter on the Makefile?

Reply:
I believe `feature_phone/tests.py` is the only Python source file missing from this list, but that's because it's almost empty (linter complains there is no module docstring). Will add the file to the linter if we write tests.